metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -595,8 +595,8 @@ def create(cls, config_dict: dict, ctx: PipelineContext) -> Source:
         return cls(ctx, config)
 
     def get_workunits(self) -> Iterable[MetadataWorkUnit]:
-        yield from self.emit_dashboard_mces()
         yield from self.emit_card_mces()
+        yield from self.emit_dashboard_mces()
```

Comment:
I'm somewhat confused - Can you explain how this change fixes missing charts?

Reply:
If we have dashboards emitted before charts then dashboards will have charts in the chart tabs. But when we go to the chart itself it will be missing by clicking on the chart. This is noticeable only when dashboard ingestion is taking too long or it fails.